test/Search_UnitTestHelpers.hpp

```diff
@@ -199,93 +193,43 @@ auto makeIntersectsBoxWithAttachmentQueries(
   auto queries_host =
       Kokkos::create_mirror_view(Kokkos::WithoutInitializing, queries);
   for (int i = 0; i < n; ++i)
-    queries_host(i) = ArborX::attach(ArborX::intersects(boxes[i]), data[i]);
+    queries_host(i) =
+        ArborX::attach(ArborX::intersects(geometries[i]), data[i]);
   Kokkos::deep_copy(exec_space, queries, queries_host);
   return queries;
 }
 
-template <typename DeviceType, int DIM = 3, typename Coordinate = float>
+template <typename DeviceType, typename Geometry>
 auto makeNearestQueries(
-    std::vector<std::pair<ArborX::Point<DIM, Coordinate>, int>> const &points,
+    std::vector<std::pair<Geometry, int>> const &geometries,
     typename DeviceType::execution_space const &exec_space = {})
 {
-  // NOTE: `points` is not a very descriptive name here. It stores both the
+  // NOTE: `geometries` is not a very descriptive name here. It stores both the
   // actual point and the number k of neighbors to query for.
```

Comment:
May improve the comment: "The individual geometry element are like pairs storing ..."